src/services/domain/roles/dto/roles.dto.result.hub.ts

```diff
@@ -21,8 +21,8 @@ export class RolesResultHub extends RolesResult {
   })
   opportunities: RolesResult[] = [];
 
-  constructor(hub: IHub, contributorID: string) {
-    super(hub.nameID, `${contributorID}/${hub.id}`, hub.displayName);
+  constructor(hub: IHub, _contributorID: string) {
```

Comment:
Feedback addressed

Reply:
I left it to avoid a chain refactor into roles.service. (See my last commit). I have removed it from everywhere now.
It was a "let's keep it just in case we need to add it back", but yeah, they were not needed. It's working